### ISSUE TITLE
Only loaded .env environment variables and not other files (with .mode and/or .local)

### ISSUE THREAD

**Issue by bameda** [storybookjs/vue-cli-plugin-storybook#93]
In my vue project, environment variables are only loaded from `.env` and the rest of file `.env{.mode}{.local}` are ignored.

There is a test project here (https://github.com/bameda/test-vue-with-storybook) to reproduce the bug.

I've created two files:
```
# .env
VUE_APP_TEST_VARIABLE={from_.env}
```

```
# .env.development
VUE_APP_TEST_VARIABLE={from_.env.development}
```
When you click `MyButton`, you obtain two different results:

- With `yarn run serve`
  ![image](https://user-images.githubusercontent.com/877218/75781530-a6ccc200-5d5d-11ea-8822-c9b2f138117d.png)
- With `yarn run storybook: serve`
  ![image](https://user-images.githubusercontent.com/877218/75781618-d4197000-5d5d-11ea-879c-81e26c3b5fda.png)

**Comment by pksunkara** [comment]
It looks a storybook bug. I did some work on storybook env loading, but it is a bit complicated. Can you try to reproduce it without using this plugin?

**Comment by bameda** [comment]
I don't now, but environment variables works properly

```bash
export VUE_APP_TEST_VARIABLE=from the system env vars
yarn run storybook:serve
```

**Comment by pksunkara** [comment]
Yup, looks like definitely a storybook bug. @ndelangen Do you know if anything related env was changed recently?

**Comment by ndelangen** [comment]
It would help to know the version this bug appears on?
